experimental/check_gpu_ecc/check_gpu_ecc.py

```diff
@@ -120,14 +182,18 @@ def check_gpu_remapped_rows_uncorrectable(ecc_d, hostname):
 def check_gpu_sram(ecc_d, hostname):
     for gpu_uuid in ecc_d["gpu_uuid"]:
         gpu_id = ecc_d["gpu_uuid"][gpu_uuid]["gpu_id"]
-        if ecc_d["gpu_uuid"][gpu_uuid]["EEUVS"] > 0:
-           print("Warning: Detected a GPU SRAM uncorrectable error for the volatile counter for GPU ID {}, please offline this node ({}), get the HPC diagnostics and submit a support request.".format(gpu_id,hostname))
-        if ecc_d["gpu_uuid"][gpu_uuid]["EEUAS"] > 0:
-           print("Warning: Detected a GPU SRAM uncorrectable error for the aggregate counter for GPU ID {}, please offline this node ({}), get the HPC diagnostics and submit a support request.".format(gpu_id,hostname))
-        if ecc_d["gpu_uuid"][gpu_uuid]["EECVS"] > SRAM_ECC_COUNTER_THRESHOLD:
-           print("Warning: Detected a large number of GPU SRAM correctable error for the volatile counter for GPU ID {}, please offline this node ({}), get the HPC diagnostics and submit a support request.".format(gpu_id,hostname))
-        if ecc_d["gpu_uuid"][gpu_uuid]["EECAS"] > SRAM_ECC_COUNTER_THRESHOLD:
-           print("Warning: Detected a large number of GPU SRAM correctable error for the aggregate counter for GPU ID {}, please offline this node ({}), get the HPC diagnostics and submit a support request.".format(gpu_id,hostname))
+        if ecc_d["gpu_uuid"][gpu_uuid]["EEUVS"] > 0 and ecc_d["gpu_uuid"][gpu_uuid]["EEUVS"] < SRAM_ECC_COUNTER_THRESHOLD:
+           print("Warning: Detected a GPU SRAM uncorrectable error for the volatile counter for GPU ID {}, please continue to monitor this node ({}),  no additional action is required at this time.".format(gpu_id,hostname))
+        if ecc_d["gpu_uuid"][gpu_uuid]["EEUAS"] > 0 and ecc_d["gpu_uuid"][gpu_uuid]["EEUAS"] < SRAM_ECC_COUNTER_THRESHOLD:
+           print("Warning: Detected a GPU SRAM uncorrectable error for the aggregate counter for GPU ID {}, please continue to monitor this node ({}), no additional action is required at this time.".format(gpu_id,hostname))
+        if ecc_d["gpu_uuid"][gpu_uuid]["EEUVS"] > SRAM_ECC_COUNTER_THRESHOLD:
+           print("Warning: Detected a large number of GPU SRAM uncorrectable error for the volatile counter for GPU ID {}, please offline this node ({}), get the HPC diagnostics and submit a support request.".format(gpu_id,hostname))
```

Comment:
What does "get the HPC diagnostics" mean?  Run azhpc-diagnostics?

Reply:
Yes